cluster-autoscaler/FAQ.md

```diff
@@ -36,6 +36,13 @@ underutilized or completely empty and then CA will delete such unneded nodes.
 * Specify requests for your pods.
 * Use PodDisruptionBudgets to prevent pods from being deleted (if needed).
 * Check if your cloud provider quota is big enough before specifying min/max settings for your node pools.
+* Do not run any additional node group autoscalers (especially these from your cloud provider).
+
+### Is Cluster Autoscaler compatible with CPU-based autoscalers. 
```

Comment:
Maybe change autoscalers to "node autoscalers"? HPA is a CPU-based autoscaler that is meant to be used with CA.